var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -100,6 +100,7 @@ class Gcc(AutotoolsPackage):
     depends_on('isl@0.15:0.18', when='@6:8.9')
     depends_on('isl@0.15:0.20', when='@9:')
     depends_on('zlib', when='@6:')
+    depends_on('libiconv')
```

Comment:
I'm not sure if this dependency is actually required, but if it's present on the system, the build picks it up, and there doesn't appear to be a way to prevent it.